homeassistant/components/ovo_energy/__init__.py

```diff
@@ -28,6 +30,13 @@
 PLATFORMS = [Platform.SENSOR]
 
 
+class OVOCoordinatorData(BaseModel):
+    """OVO Energy data."""
+
+    daily_usage: OVODailyUsage = Field(None, alias="daily_usage")
+    plan: OVOPlan = Field(None, alias="plan")
```

Comment:
```suggestion
    daily_usage: OVODailyUsage = Field(None)
    plan: OVOPlan = Field(None)
```
No need to add an alias for the same name.